### DISCUSSION TITLE
How to use react-hotkeys-hook to submit a form

### DISCUSSION THREAD

**Discussion by dakshsaraf** [react-hook-form/react-hook-form#4900] [Q&A] [answered]
I want to submit a form using the useHotKeys from react-hotkeys-hook 

I have attached a code sandbox that does not work and is not able to submit a form when a hotkey is clicked. Can this be done?
https://codesandbox.io/s/react-hoot-form-usehotkeys-009rk?file=/src/App.js

**Comment by bluebill1049** [comment, marked as answer]
This is what you are missing: 

`handleSubmit(onSubmit)()` it's a callback

**Reply by dakshsaraf** [reply]
Thanks
